public_html/storefront/model/checkout/extension.php

```diff
@@ -76,6 +79,18 @@ public function getExtensions($type)
                             );
                         }
                     }
+
+                    //filter by order subtotal limits
+                    $minOrderSubtotal = $this->config->get($row['key']."_min_order_subtotal") ? : 0;
+                    $maxOrderSubtotal = $this->config->get($row['key']."_max_order_subtotal") ? : 0;
+                    $minOrderSubtotal = $this->currency->convert($minOrderSubtotal, $this->config->get('config_currency'), $this->currency->getCode());
+                    $maxOrderSubtotal = $this->currency->convert($maxOrderSubtotal, $this->config->get('config_currency'), $this->currency->getCode());
+
+                    if ($cartSubtotal
+                        && ($cartSubtotal <= $minOrderSubtotal || ($maxOrderSubtotal && $cartSubtotal >= $maxOrderSubtotal))
```

Comment:
Why <= and >= operators here? If cart subtotal equal min or max it's a valid value for shipment

Reply:
my consideration here is  e.g the min is set to 10, and the order subtotal is 10.00, the shipper is shown. same for the max.

Reply:
look at your code. Line 92. 
By logic if order subtotal 10 and min 10 you continue then (shipping not shown). It's a bug